sensor_msgs/Image image
---
clopema_calibration/Dotmatrix dotmatrix
